SU2_CFD/src/fluid/CFluidScalar.cpp

```diff
@@ -212,13 +212,43 @@ su2double CFluidScalar::ComputeMeanSpecificHeatCp(const su2double* val_scalars)
   return mean_cp;
 }
 
+su2double CFluidScalar::ComputeEnthalpyFromT(const su2double val_temperature, const su2double* val_scalars){
+  su2double val_Enthalpy = Cp * (val_temperature - 298.15);
```

Comment:
Isn't T0 a config option? It is bad practice to hard code constants like this.

Reply:
https://ansyshelp.ansys.com/public/account/secured?returnurl=////Views/Secured/corp/v242/en/flu_th/flu_th_sec_hxfer_theory.html this is the theory related to the enthalpy equation for multicomponent gases. T_ref is always 298.15 K for this equation

Reply:
Then make it a constant, not a hard-coded value everywhere.